hawkular-nest/hawkular-nest-feature-pack/src/main/xsl/subsystem-templates/hawkular-nest-ejb3.xsl

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xalan="http://xml.apache.org/xalan" version="2.0"  exclude-result-prefixes="xalan">
+
+  <xsl:output method="xml" version="1.0" encoding="UTF-8" indent="yes" xalan:indent-amount="4" standalone="no" />
+
+  <!-- //*[local-name()='config']/*[local-name()='supplement' and @name='default'] is an xPath's 1.0
+       way of saying of xPath's 2.0 prefix-less selector //*:config/*:supplement[@name='default']  -->
+  <xsl:template match="//*[local-name()='config']/*[local-name()='supplement' and @name='default']">
+    <xsl:copy>
+      <xsl:apply-templates select="node()|comment()|@*" />
+      <!-- Copy the MDB replacement from full to default -->
+      <xsl:copy-of select="//*[local-name()='config']/*[local-name()='supplement' and @name='full']/*[local-name()='replacement' and @placeholder='MDB']"/>
```

Comment:
> I don't remember about a change from "full" to "default", though. 

`hawkular-nest-ejb3.xsl` is a copy of `hawkular-accounts-ejb3.xsl` https://github.com/hawkular/hawkular-accounts/blob/master/feature-pack/src/main/xsl/subsystem-templates/hawkular-accounts-ejb3.xsl#L31-L32

`hawkular-accounts-ejb3.xsl` was doing exactly the same thing as https://git.io/vrC6E and https://git.io/vrC61 were doing before we had `hawkular-accounts-ejb3.xsl`.

> I also don't think any other component uses MDB, so, I'd just try to remove this. One less thing to maintain.

No, a quick search for `@MessageDriven` in Hawkular components shows occurences in Alerts, BTM, avail-creator and hawkular-api. I vote for leaving `hawkular-nest-ejb3.xsl` as is.